server/entity/damage/source.go

```diff
@@ -114,3 +120,8 @@ func (SourceFireTick) ReducedByArmour() bool {
 func (SourceLava) ReducedByArmour() bool {
 	return true
 }
+
+// ReducedByArmour ...
+func (SourceProjectile) ReducedByArmour() bool {
+	return false
```

Comment:
This doesn't sound right?

Reply:
Oops, this should be true, yes. Will change this.